account_budget_activity/models/account_budget.py

```diff
@@ -601,9 +602,10 @@ def _default_period_split_lines(self):
     )
     released_amount = fields.Float(
         string='Released Amount',
-        compute='_compute_released_amount',
+#         compute='_compute_released_amount',
```

Comment:
If this method _compute_released_amount() is not used, please remove it from code. (This apply to other part)